pyfqmr/Simplify.pyx

```diff
@@ -107,7 +107,12 @@ cdef class Simplify :
         self.triangles_cpp.clear()
         self.vertices_cpp.clear()
         self.normals_cpp.clear()
+
         # Here we will need some checks, just to make sure the right objets are passed
+        # Only checks on the faces is possible, as they are necessarily integers.
+        # Maybe other checks like max(faces)==len(vertices) could be added
+        assert is_integer_array(faces), "Faces must contain only integers, check function argument order"
```

Comment:
Can we throw a proper `TypeError`?

Reply:
I'll  remove the "is_interger_array" methods then, and only check if the face array is of integer type, and throw a more specific TypeError.